lesson_10/src/library/serviceClasses/Menu.java

```diff
@@ -0,0 +1,166 @@
+package library.serviceClasses;
+
+import java.util.Scanner;
+
+public class Menu {
```

Comment:
Not sure that menu class methods should be static. You can just create an object of menu class and use its methods